contracts/src/token/erc721/extensions/enumerable.rs

```diff
@@ -180,8 +183,8 @@ impl Erc721Enumerable {
     ///
     /// # Errors
     ///
-    /// If owner address is `Address::ZERO`, then the error
-    /// [`crate::token::erc721::Error::InvalidOwner`] is returned.
+    /// * [`crate::token::erc721::Error::InvalidOwner`] - If owner address is
+    ///   `Address::ZERO`.
```

Comment:
```suggestion
    /// * [`erc721::Error::InvalidOwner`] - If owner address is `Address::ZERO`.
```
`crate::token::erc721` is already [in scope](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/497/files#diff-cd03e5643740f298f1635064125fd7a763157f711d63ef13a50387940a851e92R24), remove all `crate::token::` imports in docs